ocs_ci/deployment/deployment.py

```diff
@@ -927,12 +927,18 @@ def setup_local_storage(storageclass):
     # Get the worker nodes
     workers = get_typed_nodes(node_type="worker")
     worker_names = [worker.name for worker in workers]
-    worker_names_str = " ".join(worker_names)
-    logger.debug("Workers: %s", workers)
+    logger.debug("Workers: %s", worker_names)
 
     # Label the worker nodes
     logger.info("Labeling worker nodes")
-    run_cmd(f"oc label nodes {worker_names_str} " f"{constants.OPERATOR_NODE_LABEL}")
+    to_label = config.DEPLOYMENT.get("ocs_operator_nodes_to_label", 3)
```

Comment:
I am curious why we don't use function for labeling and taint here:
`label_and_taint_nodes`
https://github.com/red-hat-storage/ocs-ci/blob/41034ca4342ea092c91d2ebe4b877ec2585b73c4/ocs_ci/deployment/deployment.py#L176

Only one problem I see that this function is called after:
https://github.com/red-hat-storage/ocs-ci/blob/41034ca4342ea092c91d2ebe4b877ec2585b73c4/ocs_ci/deployment/deployment.py#L381

But if we will change the order and move the code:
https://github.com/red-hat-storage/ocs-ci/blob/41034ca4342ea092c91d2ebe4b877ec2585b73c4/ocs_ci/deployment/deployment.py#L370-L371

After the place where we call function for labeling I think it should be feasible to handle labeling on one place and if some differences are needed this should be handled in that function.